skorch/setter.py

```diff
@@ -0,0 +1,70 @@
+"""Setter functions for virtual params such as ``optimizer__lr``."""
+import re
+
+
+def _extract_param_name_and_group(optimizer_name, param):
```

Comment:
Does this function have any application besides for the optimizer? If not, would it not be better to have `optimizer` somewhere in the name?